action.yml

```diff
@@ -10,9 +10,14 @@ inputs:
   badge-style:
     description: Badge style generated by shields.io (one of flat, flat-square, plastic, for-the-badge, social).
     default: flat
+  badge-title:
+    description: Badge title generated by shields.io.
+    default: coverage
   report:
     description: Generate an HTML coverage report.
     default: true
+  report-path:
+    description: Custom path for an HTML coverage report.
```

Comment:
This isn't really a the report path.
It's actually the output folder where _all_ output files are created, right?

Reply:
Yes, actually it is, I just didn't want to add something new, but if you have a suggestions, I'll fix it..

Anyway, I can rename param and ENVs like:
```
output-path:
    description: Custom path for an HTML coverage report and other results file like svg badge and chart.
```

What do you think?

Reply:
fixed